common/src/main/java/org/apache/drill/common/exceptions/UserException.java

```diff
@@ -549,7 +550,12 @@ public UserException build(final Logger logger) {
       if (isSystemError) {
         logger.error(newException.getMessage(), newException);
       } else {
-        logger.info("User Error Occurred", newException);
+        String msg = "User Error Occurred";
+        if (message != null) {
+          msg += ": " + message; }
+        if (cause != null) {
```

Comment:
Wonder if the root cause (see line 540 above) may also be useful (in case it is below/different from 'cause') ?